test_calcuator.py

```diff
@@ -0,0 +1,57 @@
+"""Calculator Tests."""
+
+import unittest
+import Calculator_exercise
+
+
+class TestCalc(unittest.TestCase):
+    """Tests solving different equations"""
+
+    def setUp(self):
+        pass
+
+    def test_add(self):
```

Comment:
For all of your tests you need to make sure you test negatives and decimals as well.